buildSrc/src/main/kotlin/io.customer/android/Dependencies.kt

```diff
@@ -31,6 +31,8 @@ object Dependencies {
     const val gradleNexusPublishPlugin =
         "io.github.gradle-nexus:publish-plugin:${Versions.GRADLE_NEXUS_PUBLISH_PLUGIN}"
     const val junit4 = "junit:junit:${Versions.JUNIT4}"
+    const val junitBom = "org.junit:junit-bom:${Versions.JUNIT_BOM}"
```

Comment:
if we are adding `junitbom` do we need to specify version for `junit4` above?

Reply:
Yes because we are not modifying tests in `common-test` module in this ticket. When we work on those tests, we can surely remove this.